README.md

```diff
@@ -42,50 +42,51 @@ See <https://github.com/ansible/ansible/issues/71528> for more information.
-| kafka_java_heap                                | -Xms1G -Xmx1G                         |
-| kafka_background_threads                       | 10                                    |
-| kafka_listeners                                | PLAINTEXT://:9092                     |
-| kafka_num_network_threads                      | 3                                     |
-| kafka_num_io_threads                           | 8                                     |
-| kafka_num_replica_fetchers                     | 1                                     |
-| kafka_socket_send_buffer_bytes                 | 102400                                |
-| kafka_socket_receive_buffer_bytes              | 102400                                |
-| kafka_socket_request_max_bytes                 | 104857600                             |
-| kafka_replica_socket_receive_buffer_bytes      | 65536                                 |
-| kafka_data_log_dirs                            | /var/lib/kafka/logs                   |
-| kafka_num_partitions                           | 1                                     |
-| kafka_num_recovery_threads_per_data_dir        | 1                                     |
-| kafka_log_cleaner_threads                      | 1                                     |
-| kafka_offsets_topic_replication_factor         | 1                                     |
-| kafka_transaction_state_log_replication_factor | 1                                     |
-| kafka_transaction_state_log_min_isr            | 1                                     |
-| kafka_log_retention_hours                      | 168                                   |
-| kafka_log_segment_bytes                        | 1073741824                            |
-| kafka_log_retention_check_interval_ms          | 300000                                |
-| kafka_auto_create_topics_enable                | false                                 |
-| kafka_delete_topic_enable                      | true                                  |
-| kafka_default_replication_factor               | 1                                     |
-| kafka_group_initial_rebalance_delay_ms         | 0                                     |
-| kafka_zookeeper_connect                        | localhost:2181                        |
-| kafka_zookeeper_connection_timeout             | 6000                                  |
-| kafka_bootstrap_servers                        | localhost:9092                        |
-| kafka_consumer_group_id                        | kafka-consumer-group                  |
-
-See [log4j.yml](./defaults/main/002-log4j.yml) for detailled  
+| Variable                                       | Default                          |
+| ---------------------------------------------- | -------------------------------- |
+| kafka_download_base_url                        | <https://dlcdn.apache.org/kafka> |
+| kafka_download_validate_certs                  | yes                              |
```

Comment:
The Lets Encrypt certificate used by this server can cause cert expiration issues depending on the version of OpenSSL and cacerts that are installed on a machine. Setting this to `no` will workaround certificate problems when the task attempts to download the tar package, CentOS 7 is one such OS where this is an issue. cURL works fine but wget does not.